src/averageAges.js

```diff
@@ -38,6 +57,28 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const womenOfPeople = people.filter(x => x.sex === 'f');
+
+  const callback = (total, next) => {
```

Comment:
This 'callback' function looks the same as declared below and does the same calculation. Consider defining it globally to avoid repetition.